### DISCUSSION TITLE
Coding

### DISCUSSION THREAD

**Discussion by irhrhd** [Prodigy-Hacking/ProdigyMathGameHacking#618] [(A) Anything]
Which do i need to learn the most to be able to help script the hacks and rescript them
Web Development
Data Science
Computer Science
Machine Learning
Web Design
Game Development
Mobile Development
Data Visualization

**Comment by PatheticMustan** [comment]
The only two things you need is probably web development and web design.

**Reply by ghost** [reply]
Where should I learn to do web development and wed design

**Reply by ArcerionDev** [reply]
your brain

**Reply by ArcerionDev** [reply]
https://google.com/search?q=coding+courses

**Reply by ghost** [reply]
> https://google.com/search?q=coding+courses

Wow very helpful. Not really.

**Reply by ghost** [reply]
https://learntocodewith.me/posts/code-for-free/

**Comment by ghost** [comment]
Thx
